libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -140,16 +140,17 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @Increment: 50
     // @User: Advanced
     AP_GROUPINFO("ARM_MAH", 19, AP_BattMonitor_Params, _arming_minimum_capacity, 0),
+#endif // HAL_BUILD_AP_PERIPH
 
     // 20 was BUS
 
     // @Param: OPTIONS
     // @DisplayName: Battery monitor options
     // @Description: This sets options to change the behaviour of the battery monitor
     // @Bitmask: 0:Ignore DroneCAN SoC, 1:MPPT reports input voltage and current, 2:MPPT Powered off when disarmed, 3:MPPT Powered on when armed, 4:MPPT Powered off at boot, 5:MPPT Powered on at boot, 6:Send resistance compensated voltage to GCS
+    // @Bitmask{AP_Periph}: 7:Internal Use only. Do not broadcast this instance on the CAN bus.
```

Comment:
I wonder what "Internal" means in this case.  Perhaps we could make this clearer to more users?